porcupine/actions.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Callable, Union
+
+from porcupine.tabs import FileTab
+
+action_availability_callback = Callable[[], bool]
```

Comment:
I think these type aliases are no longer needed.

Reply:
It's used in a few places, or you mean it's just unnecessary to define them?

Reply:
It's unnecessary to define them. Understanding `Callable[[], bool]` is much easier than understanding `action_availability_callback`. The type aliases were more useful with `list[action_availability_callback]`, but we can simplify since we no longer have a list of callbacks.

Reply:
fair enough

Reply:
253fdca